SPARQLWrapper/Wrapper.py

```diff
@@ -905,54 +909,55 @@ def _query(self):
         request = self._createRequest()
 
         try:
-            if self.timeout:
+            if self.timeout is not None:
```

Comment:
This looks like it can go into a separate PR.

Reply:
934e334